demos/src/scss/_demos.scss

```diff
@@ -48,10 +48,9 @@ body:before {
 }
 
 .o-grid-container {
-	background: lighten(#f6e9d8, 8.5%);
+	background: #EAE8DF;
 }
```

Comment:
No major opinion here, I think both look ok. The row colouring would make debugging easier, but can easily be added when doing so, @triblondon any reason for changing them?